### DISCUSSION TITLE
Github copilot not working as expected

### DISCUSSION THREAD

**Discussion by K9Developer** [community/community#10988] [Copilot Conversations]
It almost doesn't complete my lines, for example, i wrote
`# Create a function that adds 2 numbers together`
but it didnt do anything, so i tried your example with python:
```
import datetime

def parse_expenses(expenses_string):
    """Parse the list of expenses and return the list of triples (date, value, currency).
    Ignore lines starting with #.
    Parse the date using datetime.
    Example expenses_string:
        2016-01-02 -34.01 USD
        2016-01-03 2.59 DKK
        2016-01-03 -2.72 EUR
    """
```
still nothing.
(I used vscode) I tried to do it in pycharm and it works way better.

**Comment by davecheney** [comment]
> why did you change the GitHub copilot icon?

what was it changed too?

**Reply by K9Developer** [reply]
I had to reinstall it for i to update the icon
